src/test/java/friday/storage/JsonSerializableFridayTest.java

```diff
@@ -5,49 +5,52 @@
-            "JsonSerializableMasteryCheckBookTest");
-    private static final Path TYPICAL_STUDENTS_FILE = TEST_DATA_FOLDER.resolve("typicalStudentsAddressBook.json");
-    private static final Path INVALID_STUDENT_FILE = TEST_DATA_FOLDER.resolve("invalidStudentAddressBook.json");
-    private static final Path DUPLICATE_STUDENT_FILE = TEST_DATA_FOLDER.resolve("duplicateStudentAddressBook.json");
-
-    /*
-    @Test
-    public void toModelType_typicalStudentsFile_success() throws Exception {
-        JsonSerializableAddressBook dataFromFile = JsonUtil.readJsonFile(TYPICAL_STUDENTS_FILE,
-                JsonSerializableAddressBook.class).get();
-        AddressBook addressBookFromFile = dataFromFile.toModelType();
-        AddressBook typicalStudentsAddressBook = TypicalStudents.getTypicalAddressBook();
-        assertEquals(addressBookFromFile, typicalStudentsAddressBook);
-    }
-
-    @Test
-    public void toModelType_invalidStudentFile_throwsIllegalValueException() throws Exception {
-        JsonSerializableAddressBook dataFromFile = JsonUtil.readJsonFile(INVALID_STUDENT_FILE,
-                JsonSerializableAddressBook.class).get();
-        assertThrows(IllegalValueException.class, dataFromFile::toModelType);
-    }
+//    private static final Path TEST_DATA_FOLDER = Paths.get("src", "test", "data",
+//            "JsonSerializableFridayTest");
+//    private static final Path TYPICAL_STUDENTS_FILE = TEST_DATA_FOLDER.resolve("typicalStudentFriday.json");
+//    private static final Path INVALID_STUDENT_FILE = TEST_DATA_FOLDER.resolve("invalidStudentFriday.json");
+//    private static final Path DUPLICATE_STUDENT_FILE = TEST_DATA_FOLDER.resolve("duplicateStudentFriday.json");
+//
+//
+//    @Test
+//    public void toModelType_typicalStudentsFile_success() throws Exception {
+//        JsonSerializableFriday dataFromFile = JsonUtil.readJsonFile(TYPICAL_STUDENTS_FILE,
+//                JsonSerializableFriday.class).get();
+//        Friday fridayFromFile = dataFromFile.toModelType();
+//        Friday typicalStudentsAddressBook = TypicalStudents.getTypicalFriday();
+//        assertEquals(fridayFromFile, typicalStudentsAddressBook);
+//    }
+//
+//    @Test
+//    public void toModelType_invalidStudentFile_throwsIllegalValueException() throws Exception {
+//        JsonSerializableFriday dataFromFile = JsonUtil.readJsonFile(INVALID_STUDENT_FILE,
+//                JsonSerializableFriday.class).get();
+//        assertThrows(IllegalValueException.class, dataFromFile::toModelType);
+//    }
+//
+//    @Test
+//    public void toModelType_duplicateStudents_throwsIllegalValueException() throws Exception {
+//        JsonSerializableFriday dataFromFile = JsonUtil.readJsonFile(DUPLICATE_STUDENT_FILE,
+//                JsonSerializableFriday.class).get();
+//        assertThrows(IllegalValueException.class, JsonSerializableFriday.MESSAGE_DUPLICATE_STUDENT,
+//                dataFromFile::toModelType);
+//    }
```

Comment:
As part of the code quality standard, I think we should remove commented out code